Assets/HoloToolkit-Examples/GameControllerManipulator/GameControllerManipulatorTest.unity

```diff
@@ -0,0 +1,524 @@
+%YAML 1.1
```

Comment:
Sorry for the delayed response. Is this a Unit test type scene?
If so then please move it under the Tests folder under Input.